otherlibs/threads/unix.ml

```diff
@@ -1124,28 +1124,38 @@ let find_proc_id fun_name proc =
   with Not_found ->
     raise(Unix_error(EBADF, fun_name, ""))
 
+let process_in_pid inchan =
+  find_proc_id "close_process_in" (Process_in inchan)
```

Comment:
The string literal is displayed in case of error and should correspond to the calling function in each case, so ideally it should be `"process_in_pid"` when calling `process_in_pid`, and `"close_process_in"` when calling `close_process_in` (same for the other functions).

Reply:
Gotcha.